### PR TITLE
Updating the docs to get rid of a few typos

### PR DESCRIPTION
**Pull request by gperez-10pines**
While reading the docs I spotted a couple of typos, so I took the chance to run a spell-checker and get rid of them.

### CODE REVIEW

File: worlds/tutorial2.js [hunk 1]
```diff
@@ -31,7 +31,6 @@ export function init(Constants) {
                 shadow: true,
                 layers: ["walk"],
                 translation:[0, -1.7, 0],
-                shadow: true,
```
Comment (gperez-10pines) [inline worlds/tutorial2.js:34]: The variable was previously declared

File: docs/Tutorial.md [hunk 2]
```diff
@@ -368,7 +368,7 @@ http://localhost:9684/?world=tutorial2
 
 Here is an image of the world we will be constructing:
 
-![Tutorial 2](./assets/tutorial2.png)
+![Tutorial 2](./docs/assets/Tutorial2.png)
```
Comment (gperez-10pines) [inline docs/Tutorial.md:371]: You can see this here:
[broken image](https://github.com/croquet/microverse-builder/blob/main/docs/Tutorial.md#tutorial-2)

**Comment by yoshikiohshima** [comment]
Thank you!  We are not ready to take  in a pull request directly into this generated repository, but if you don't mind, let me incorporate those changes into our code base.

**Comment by gperez-10pines** [comment]
Sure, let's just go ahead and close the PR.

**Pull request closed without merging**